src/main/java/frc/robot/commands/elevator/GetMaxHeight.java

```diff
@@ -0,0 +1,45 @@
+package frc.robot.commands.elevator;
+import edu.wpi.first.wpilibj.smartdashboard.SmartDashboard;
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.constants.ElevatorConstants;
+import frc.robot.subsystems.Elevator;
+
+public class GetMaxHeight extends CommandBase {
+  Elevator m_elevator; 
+  /**
+   * Calibrate the elevator by resetting the encoder after it hits the bottom
+   * limit switch. 
```

Comment:
You might want to make it clear geting the max hieght is not the only part of the calibration prosses. Also why is this spread across multiple lines.

Reply:
Multiple lines for jdoc comments is fine, it shows up as one line when hovering and it keeps one line from being too long.
But, I would rename this to CalibrateMaxHeight() because currently it isn't clear that it's not just getting the max height, it is actually moving the robot. Don't want to cause any confusion and accidentally move the robot, though it's unlikely.